0_portfolio_input_check_functions.R

```diff
@@ -864,16 +864,18 @@ create_id_columns <- function(portfolio, portfolio_type) {
 
 get_and_clean_total_fund_list_data <- function() {
   total_fund_list <- read_rda("data/total_fund_list.rds")
-  
-  
+
+
   total_fund_list
 }
 
 get_and_clean_currency_data <- function() {
-  currencies <- read_rda("data/currencies.rda")
-  
+  currencies <- readRDS("data/currencies.rds")
+
   currencies <- set_currency_timestamp(currencies)
-  
+
+  if (all(currencies$exchange_rate) == 1){print("currency data temporary. todo: update")}
```

Comment:
I don't understand `all(currencies$exchange_rate)`.... presumably `currencies$exchange_rate` is a vector of values, so `all(currencies$exchange_rate)` evaluates to `TRUE` if all of the values in it are >0, and `TRUE` is automatically coerced to `1` when it's compared with ` == 1`?

Reply:
Hmm... basically I created a fake column in the df for 2020Q4 and filled it with 1s, so wanted to make sure that this eventually gets replaced with the actual values. Any good ideas for this?

Reply:
So I struggled with this, and couldn't create an alternate file that included an extra column that could be opened by both calls to it (as discussed in Slack). We can either keep both files in here until this is resolved, or something else that I'm not sure about

Reply:
7d0f382 restores data/currencies.rda. Having it and not using it seems safer than removing it.

Reply:
> I created a fake column in the df for 2020Q4 and filled it with 1s ...

I thinks it's more natural to use `NA` because it's R's representation of missing values. I would be more specific and fill it with the specific type of `NA` that fits the data you expect to eventually get in there:


``` r
data <- tibble::tibble(x = 1:3)
data$y <- NA_integer_
data
#> # A tibble: 3 x 2
#>       x     y
#>   <int> <int>
#> 1     1    NA
#> 2     2    NA
#> 3     3    NA

if (all(is.na(data$y))) {
  warning("TODO: update.")
}
#> Warning: TODO: update.
```

<sup>Created on 2021-02-17 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>